### PR TITLE
Reusable GET method for private API

### PR DESCRIPTION
**Pull request by MartinCaparros**
I create an axios instance and a interceptor so we can check the headers authorization before doing the request, and also create an reusable GET request that takes 3 parameters (url, id, params).

User Story: https://alkemy-labs.atlassian.net/browse/OT91-68

### CODE REVIEW

File: src/Services/privateApiService.js [hunk 1]
```diff
@@ -6,14 +6,24 @@ const config = {
     }
 }
 
+const privateInstance = axios.create({
+    baseURL: 'http://ongapi.alkemy.org/public/api',
+});
+
+privateInstance.interceptors.request.use((config)=> {
+    config.headers.Authorization = localStorage.getItem('token');
```
Comment (saralavanchy) [inline src/Services/privateApiService.js]: please reuse the method created by @ArielSurco on the user story [Crear método que devuelva el encabezado de seguridad para ser reutilizado en las peticiones a los metodos privados de la API](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT91/boards/145?modal=detail&selectedIssue=OT91-69)

Comment (saralavanchy) [reply src/Services/privateApiService.js]: the function is on this file: getAuthorizationHeader

File: src/Services/privateApiService.js [hunk 2]
```diff
@@ -6,14 +6,24 @@ const config = {
...
+    config.headers.Group = 91;
+    return config
+})
+
+export const privateGet = async (url,id,params={}) => {privateInstance.get(`${url}${id?`/${id}`:''}`,{params})};
```
Comment (saralavanchy) [inline src/Services/privateApiService.js]: the id can be null, you should intercept that adding ? on the parameters. Sometimes its good to have all on the same line and sometimes not, this is a place where its not. Maybe a constant would help some programmers to understand better this functionallity
```
=> {
    const idPlaceholder = id ? `/${id}` : '';
    privateInstance.get(`${url}${idPlaceholder}`, {params})
};
```

File: src/Services/privateApiService.js [hunk 3]
```diff
@@ -6,14 +6,24 @@ const config = {
     }
 }
 
+const privateInstance = axios.create({
```
Comment (saralavanchy) [inline src/Services/privateApiService.js]: axiosInstance maybe its a better name. Then if you have a privateInstance of AWS you can call it awsInstance, and so on. privateInstance its to generic. Also, its call private instance and the baseURl says public.

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Services/privateApiService.js (hunk 1 above)
Comment (MartinCaparros) [reply src/Services/privateApiService.js]: Done

File: src/Services/privateApiService.js (hunk 2 above)
Comment (MartinCaparros) [reply src/Services/privateApiService.js]: Ok, I already changed it

File: src/Services/privateApiService.js (hunk 3 above)
Comment (MartinCaparros) [reply src/Services/privateApiService.js]: Done

File: src/Services/privateApiService.js [hunk 4]
```diff
@@ -6,16 +6,31 @@ const config = {
...
+axiosInstance.interceptors.request.use((config)=> {
+    config.headers.Authorization = getAuthorizationHeader();
+    config.headers.Group = 91;
+    return config
+})
+
+export const privateGet = async (url,id,params={}) => {
+    const idPlaceholder = id ? `/${id}` : '';
+    axiosInstance.get(`${url}${idPlaceholder}`, {params})
+};
```
Comment (saralavanchy) [inline src/Services/privateApiService.js:23]: I think that you should return the get's result

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Services/privateApiService.js (hunk 4 above)
Comment (MartinCaparros) [reply src/Services/privateApiService.js:23]: Perfect, ill change it.

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**